.github/workflows/backend-visualization.yml

```diff
@@ -1,4 +1,4 @@
-name: KFP backend visualization tests
```

Comment:
would suggest changing it back to `KFP Backend Visualization Tests`, to keep the `KFP` suffix name convention consistent across all workflow names. For instance: https://github.com/kubeflow/pipelines/blob/master/.github/workflows/presubmit-backend.yml#L1

Also to be able to differentiate between KFP and kfp-tekton workflows: https://github.com/kubeflow/pipelines/blob/master/.github/workflows/backend.yml#L1

Reply:
Makes sense.
Done.